BlazarUI/app/scripts/actions/buildActions.js

```diff
@@ -34,7 +34,7 @@ function getModule() {
   const modulesPromise = branchModules.fetch();
 
   modulesPromise.done( () => {
-    gitInfo.moduleId = _.find(branchModules.data, (m) => {
+    gitInfo.moduleId = find(branchModules.data, (m) => {
```

Comment:
I know was just wondering if we need to use `underscore` for functions that already exist natively on object prototypes in modern browsers.